src/mscorlib/src/System/Threading/EventWaitHandle.cs

```diff
@@ -176,7 +176,7 @@ private static OpenExistingResult OpenExistingWorker(string name, EventWaitHandl
 #else
             if (name == null)
             {
-                throw new ArgumentNullException(nameof(name), SR.ArgumentNull_WithParamName);
```

Comment:
`ArgumentNull_WithParamName` resource string can be deleted now as well since it is no longer used.

Reply:
done